tests/CMakeLists.txt

```diff
@@ -228,6 +235,7 @@ set(LVGL_TEST_OPTIONS_TEST_DEFHEAP
     ${LVGL_TEST_OPTIONS_TEST_COMMON}
     -DLVGL_CI_USING_DEF_HEAP
     -DLV_MEM_SIZE=2097152
+    -fsanitize=address

```

Comment:
I intentionally left it out on `DEFHEAP`, because I don't think AddressSanitizer is nearly as useful when it doesn't also control heap allocations.

Reply:
I usually use with `LV_MEM_CUSTOM=0` and still see ASAN errors.  So I thought why not to use it here too.

Reply:
Interesting. If it helps, let's definitely keep it.